numba/np/arraymath.py

```diff
@@ -785,42 +844,9 @@ def array_argmax(arr, axis=None):
         def array_argmax_impl(arr, axis=None):
             return flatten_impl(arr)
     else:
-        _check_is_integer(axis, "axis")
```

Comment:
This code now lives in `build_argmax_or_argmin_with_axis_impl`, unchanged otherwise.

Reply:
Confirmed. Thanks.